pkg/envoy/rds/route/route_config.go

```diff
@@ -101,13 +105,13 @@ func BuildIngressConfiguration(ingress []*trafficpolicy.InboundTrafficPolicy) *x
 }
 
 // BuildOutboundMeshRouteConfiguration constructs the Envoy construct (*xds_route.RouteConfiguration) for the given outbound mesh route configs
-func BuildOutboundMeshRouteConfiguration(portSpecificRouteConfigs map[int][]*trafficpolicy.OutboundTrafficPolicy) []*xds_route.RouteConfiguration {
+func BuildOutboundMeshRouteConfiguration(outbound *trafficpolicy.OutboundMeshTrafficPolicy) []*xds_route.RouteConfiguration {
```

Comment:
Why is the signature of this method being changed?

Reply:
+1. also, this change isn't a part of the retry code. If at all you need to make this change it should be a separate PR